### PR TITLE
Enable Split io A/B testing

### PR DESCRIPTION
**Pull request by MounirDhahri**
### Description

- Enable Split io A/B testing

### PR Checklist (tick all before merging)

- [x] I have not stored any sensitive keys/values/information in `Echo.json5` and in CI, or I have not changed anything in that file.

### CODE REVIEW

**Comment by MounirDhahri** [comment]
@brainbicycle going to merge this after Pierluca's approval 🟢

**Review by rajsam003 — APPROVED** [review]
LGTM

**Pull request merged**
